test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -142,9 +142,16 @@ def open_add_new_account_popup(self, attempt: int = 2):
             else:
                 raise
 
-    def open_add_account_popup(self):
+    def open_add_account_popup(self, attempt: int = 2):
         self._add_account_button.click()
-        return AccountPopup().wait_until_appears()
+        try:
+            return AccountPopup().wait_until_appears()
+        except AeertatioError as err:
```

Comment:
Is this a typo? `Aeertatio` doesn't seem like a real word :laughing:

Reply:
ohh, thanks a lot @jrainville, fixed